samtranslator/internal/schema_source/aws_serverless_graphqlapi.py

```diff
@@ -152,7 +152,19 @@ class Cache(BaseModel):
     TransitEncryptionEnabled: Optional[PassThroughProp]
 
 
+class Defaults(BaseModel):
+    # properties common for Resolvers and Functions
+    Runtime: Optional[Runtime]
+    MaxBatchSize: Optional[PassThroughProp]
+    Sync: Optional[Sync]
+    # only Resolver properties
+    Resolvers: Optional[Resolver]
+    # only Function properties
+    Functions: Optional[Function]
```

Comment:
What does it mean to have a default function?

Reply:
I means, we can reference here all Function properties

Reply:
Why not? Even properties like `CodeUri` or `InlineCode` have a use case to be declared in `Defaults`. If a customer uses Lambda Resolvers mostly, they might want to override the code we provide by default, but they still might want to have the same code for all Resolvers and Functions.

Reply:
Restricting properties as discussed